### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by vxsx**
pretty much this https://github.com/aldryn/aldryn-boilerplate-bootstrap3/pull/166

### CODE REVIEW

**Comment by lory87** [comment]
lgtm

File: .travis.yml [hunk 1]
```diff
@@ -15,8 +17,7 @@ env:
     - secure: Nwqm4GQfjZ52MvcQZ/mP068RapB6Jg3sMVmitqsYwUmZsC9xDdmjkJlv3ViWmtZYcvYvQKT7+/iZ4+NjVvYcw7n2OY2vs9F/Vf+XuXuiQEQ9Ouy5im8vWM+1xxzfzPe0tVIWDIApx2j94G+CnKMebwCUeaPeRS1H8BXClPMVkeo=
 
 before_install:
-  - gem install scss-lint
-  - npm install -g npm
+  - npm install -g npm@2
```
Comment (FinalAngel) [inline .travis.yml:20]: does this boilerplate require latest node version?

**Comment by FinalAngel** [comment]
nice work

File: .travis.yml (hunk 1 above)
Comment (vxsx) [inline .travis.yml:20]: it should be possible to run on all of them >= 0.12.7

**Pull request merged**